src/System.Runtime.Loader/tests/DefaultContext/DefaultLoadContextTest.cs

```diff
@@ -59,23 +59,38 @@ public static void LoadInDefaultContext()
             Init();
 
            // This will attempt to load an assembly, by path, in the Default Load context via the Resolving event
-           var assemblyName = "System.Runtime.Loader.Noop.Assembly";
+           var assemblyNameStr = "System.Runtime.Loader.Noop.Assembly";
 
             AssemblyLoadContext.Default.Resolving += ResolveAssembly;
-            var assemblyExpected = AssemblyLoadContext.Default.LoadFromAssemblyName(new AssemblyName(assemblyName));
+            var assemblyName = new AssemblyName(assemblyNameStr);
+
+            // This will invoke the resolution via VM requiring to bind using the TPA binder
+            var assemblyExpectedFromLoad = Assembly.Load(assemblyName);
+
+            // We should have successfully loaded the assembly in default context.
+            Assert.NotNull(assemblyExpectedFromLoad);
+
+            // And make sure the simple name matches
+            Assert.Equal(assemblyExpectedFromLoad.GetName().Name, assemblyNameStr);
```

Comment:
Nit: the arguments here should be reversed... the first argument is "expected", the second is "actual"